sdks/fabric/interoperation-node-sdk/src/AssetManager.ts

```diff
@@ -23,7 +23,7 @@ const logger = log4js.getLogger("InteroperableHelper");
 function createAssetExchangeAgreementSerialized(assetType, assetID, recipientECert, lockerECert)
 {
     const assetExchangeAgreement = new assetLocksPb.AssetExchangeAgreement();
-    assetExchangeAgreement.setAssettype(assetType);
+    assetExchangeAgreement.setType(assetType);
```

Comment:
Can you revert this change? I think your fork may not have been synced with latest code from main, may be sync it and then you will be able to build without modifying this.
Same for other places with similar change